### PR TITLE
Add red user to seeds

### PR DESCRIPTION
**Pull request by redmvc**

### CODE REVIEW

**Review by Throne3d — APPROVED** [review]

File: db/seeds.rb [hunk 1]
```diff
@@ -34,6 +34,11 @@
   username: 'Reader', password: 'reader', email: "dummy7@example.com", role_id: Permissible::READONLY,
 )
 
+User.create!(
+  username: 'red', password: 'red', email: "dummy8@example.com", role_id: Permissible::MOD,
+  layout: 'starrydark', ignore_unread_daily_report: true, created_at: "2024-11-30 15:34:00",
```
Comment (teceler) [inline db/seeds.rb:39]: I'm not sure you want the created_at in there? otherwise looks good tho

Comment (Throne3d) [reply db/seeds.rb:39]: Seems like the other seeds (except reader user) have created_at, interestingly 🤔

Comment (teceler) [reply db/seeds.rb:39]: huh, so they do! i wonder why we did that. nvm then

Comment (Throne3d) [reply db/seeds.rb:39]: no idea tbh, it's weird. cool!

**Pull request merged**